pkg/resourcemapping/orm_client_manager.go

```diff
@@ -66,36 +70,54 @@ type OwnerResources struct {
 	ControllerObj *unstructured.Unstructured
 	//---  V2 or V1
 	// mapping of owner and the owner path for a given source/owned path
-	OwnerResources map[string][]devopsv1alpha1.ResourcePath
-	ErrorMsg       error
+	OwnerResourcesMap map[string][]devopsv1alpha1.ResourcePath
+	isV1ORM           bool
+}
+
+// This helper function will determine if the owner resources are found in orm V1, since we are returning the source/owned obj resourcepaths if it cannot
+// locate owner paths. This will determine to check and log if it actually found the owners resources from v1 orm
+func ownerResourcesFound(ownedObj *unstructured.Unstructured, allOwnerResourcePaths map[string][]devopsv1alpha1.ResourcePath) bool {
+	for _, resourcePaths := range allOwnerResourcePaths {
+		for _, resourcePath := range resourcePaths {
+			if ownedObj.GetKind() != resourcePath.ObjectReference.Kind {
+				return true
+			}
+		}
+	}
+	return false
 }
 
 // Return the owner resources to modify for a given source resource path
 // to change the resources based an action recommendation from the Turbo server.
 func (manager *ORMClientManager) GetOwnerResourcesForSource(ownedObj *unstructured.Unstructured,
 	ownerReference discoveryutil.OwnerInfo, paths []string) (*OwnerResources, error) {
 	var err error
-	var owned corev1.ObjectReference
-	owned = corev1.ObjectReference{
+	var owned corev1.ObjectReference = corev1.ObjectReference{
```

Comment:
nit, move this out to a small helper function. This is repeated in several places in this file and the operator_resource_mapping.go